CampaignConfig/mdc2020_cry.ini

```diff
@@ -68,27 +68,27 @@ file_pattern_1 = %.fcl
 
 [job_type mu2e_cvmfs_cry_production_jobtype]
 launch_script = fife_launch
-parameters = [["-c ", "/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020n/link/Production/CampaignConfig/mdc2020_cry.cfg"]]
+parameters = [["-c ", "/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020u/Production/CampaignConfig/mdc2020_cry.cfg"]]
 output_file_patterns = %.art
 recoveries = [["proj_status",[["-Oglobal.override_dataset=','%(dataset)s"],["--stage ","recovery"]]]]
 
 [job_type generate_fcl_cvmfs_cry_jobtype]
 launch_script = fife_launch
-parameters = [["-c ", "/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020n/link/Production/CampaignConfig/mdc2020_cry.cfg"]]
+parameters = [["-c ", "/cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020u/Production/CampaignConfig/mdc2020_cry.cfg"]]
 output_file_patterns = %.fcl
 
 [job_type mu2e_local_cry_jobtype]
 launch_script = fife_launch
-parameters = [["-c ", "/mu2e/app/users/srsoleti/workDir/Production/CampaignConfig/mdc2020_cry.cfg"]]
+parameters = [["-c ", "/mu2e/app/users/sophie/ProductionManager/Production/CampaignConfig/mdc2020_cry.cfg"]]
```

Comment:
Can this  and the other /users/sophie scripts be migrated to cvmfs as well?   Longer-term, it's clear we need an indirection here to support development from private areas.  That may need to get integrated with MUSE.

Reply:
So the "local" stages would also be CVMFS? There was a proposed solution by Ray I added as a nother issue: https://github.com/Mu2e/Production/issues/214 would that help i.e. having the workingDir on mu2epro?